express-app/app.models.js

```diff
@@ -61,3 +62,29 @@ exports.fetchComments = (article_id) => {
     return rows;
   });
 };
+
+exports.postNewComment = ({ article_id, author, body }) => {
+  if (!author || !body) {
+    return Promise.reject({ status: 400, msg: 'Bad Request' });
+  }
+  if (body.length < 1) {
+    return Promise.reject({ status: 400, msg: 'Bad Request' });
+  }
+  const query = format(
+    'INSERT INTO comments (article_id, author, body) VALUES (%L, %L, %L) RETURNING *;',
+    article_id,
+    author,
+    body
```

Comment:
This syntax is not quite correct for the SQL query. No need to use format. You need to use the $ syntax for the values in the string and to pass an array with those variables within it as the second argument to `db.query()`. This should hopefully pass your 201 test.